include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -299,12 +299,13 @@ typedef struct OrtTensorRTProviderOptions {
 /// </summary>
 typedef struct OrtOpenVINOProviderOptions {
 #ifdef __cplusplus
-  OrtOpenVINOProviderOptions() : device_type{}, enable_vpu_fast_compile{}, device_id{}, num_of_threads{} {}
+  OrtOpenVINOProviderOptions() : device_type{}, enable_vpu_fast_compile{}, device_id{}, num_of_threads{}, use_compiled_network{} {}
 #endif
   const char* device_type;                // CPU_FP32, GPU_FP32, GPU_FP16, MYRIAD_FP16, VAD-M_FP16 or VAD-F_FP32
   unsigned char enable_vpu_fast_compile;  // 0 = false, nonzero = true
   const char* device_id;
   size_t num_of_threads;  // 0 uses default number of threads
+  unsigned char use_compiled_network; // 0 = false, nonzero = true
```

Comment:
Why is 'unsigned char' used for storing numerical values? why not something like int?

Reply:
Ideally, I would have preferred to use a bool here. since, this is C-API and The C99 standard for C language supports bool variables but a header file “stdbool.h” must be additionally included to use bool in C. 

unsigned char vs int.
I think Ryan used a unsigned char to store numerical values. It might be that char is used for optimal data size when compared to int and a char would get promoted to an int in a if statement anyway.